<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>employe</class>
 <widget class="QDialog" name="employe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>701</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>gestion employe</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>891</width>
       <height>611</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>afficher employe</string>
      </attribute>
      <widget class="QTableView" name="tableView_employe">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>801</width>
         <height>491</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>331</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="tri_id">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>tri par id</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tri_grade">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>tri par grade</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tri_nom">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>tri par nom</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>mailing list</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>ajouter employe</string>
      </attribute>
      <widget class="QLineEdit" name="nomPrenom">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="id">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="cin">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>150</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="adresse">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>210</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="tel">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>270</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="salaire">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>320</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="grade">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>370</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="login">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>430</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="email">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>490</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom et prenom:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>id:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>150</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>cin:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>210</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>adresse:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>280</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>tel:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>330</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>salaire:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>380</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>grade:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>440</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>login:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>500</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>email:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>180</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>modifier employe</string>
      </attribute>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>id:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>salaire:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>170</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>grade:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_id">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_salaire">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>100</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_grade">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>170</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>130</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>supprimer employe</string>
      </attribute>
      <widget class="QLineEdit" name="idSup">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>id:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer_employe">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>170</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>gestion fonction</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>841</width>
       <height>651</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>afficher fonction</string>
      </attribute>
      <widget class="QTableView" name="tableView_fonction">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>751</width>
         <height>431</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>ajouter fonction</string>
      </attribute>
      <widget class="QLineEdit" name="id_fonction">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>120</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="caract">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>180</y>
         <width>421</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="nb_heure">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>340</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="diplome">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>400</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>id:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>120</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>nom:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>230</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>caractéristiques:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>350</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nb_heure:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>410</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>diplome:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter_fonction">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>440</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>modifier fonction</string>
      </attribute>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>id:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>120</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>nom:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>190</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>caractéristiques:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_idf">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="text_nomf">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="text_caractf">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>190</y>
         <width>341</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier_fonction">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>360</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_11">
      <attribute name="title">
       <string>supprimer fonction</string>
      </attribute>
      <widget class="QLineEdit" name="idsupf">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="idf_sup">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>50</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer_fonction">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>130</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_12">
      <attribute name="title">
       <string>statistiques</string>
      </attribute>
      <widget class="QCustomPlot" name="customPlot" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>711</width>
         <height>391</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>accès par carte</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
